quickstart/README.md

```diff
@@ -51,40 +51,39 @@ The express install script will do quite a few things to get you bootstrapped.
     1. create a full suite of PKI
     1. create a config file for the controller
     1. create a config file for an edge router
-1. download the latest distribution of ziti from github.com/openziti/ziti/releases
+1. download the latest distribution of Ziti from github.com/openziti/ziti/releases
 1. unzip the distribution
-1. start the `ziti-controller` and `ziti-router` executables
-1. the `ziti-controller` should now be exposed on https://$(hostname):1280
+1. start the controller and router in the background
```

Comment:
please revert. don't tell someone what to do like this. i routinely do NOT run it in the background

Reply:
I believed this was a description of what expressInstall() does, not what the user should do. I may have assumed incorrectly that expressInstall() runs them in the background or skipped a step. Now that expressInstall is documented in more detail in ziti-doc, should we delete this file, or perhaps link to the website to avoid keeping this one up to date?